documentation/assemblies/overview/assembly-metrics-overview.adoc

```diff
@@ -16,22 +16,23 @@ Example metrics configuration files are provided with Strimzi.
 
 Distributed tracing complements the gathering of metrics data by providing a facility for end-to-end tracking of messages through Strimzi.
 
+Cruise Control is used to monitor the workload of Kafka clusters, but it also automates the process of rebalancing. 
```

Comment:
I'm not sure I would word it like this. We do not support any other use cases than rebalancing. This suggests we do.

Reply:
Agree. We support the rebalancing feature only but of course, CC has to monitor the cluster workload for doing it.
Using "automates" sounds that it does rebalancing automatically while it's not supported, the user has to interact with CC using `KafkaRebalance` resource; there is no automatic rebalancing done by CC without user request/interaction.

Reply:
How about:

> Cruise Control provides support for rebalancing of Kafka clusters based on workload data.

Reply:
+1